stdlib/public/core/UnsafeRawPointer.swift

```diff
@@ -455,12 +455,27 @@ public struct UnsafeRawPointer: _Pointer {
   /// - Returns: A new instance of type `T`, read from the raw bytes at
   ///   `offset`. The returned instance isn't associated
   ///   with the value in the range of memory referenced by this pointer.
+  @inlinable
   @_alwaysEmitIntoClient
   public func loadUnaligned<T>(
     fromByteOffset offset: Int = 0,
     as type: T.Type
   ) -> T {
     _debugPrecondition(_isPOD(T.self))
+#if arch(x86_64)
```

Comment:
I would assume that this also affects i386 and armv7, which both have 16B load forms that require alignment. Given that it's optimized away, maybe drop the condition entirely?